stack_definition.json

```diff
@@ -415,7 +417,71 @@
 				}
 			],
 			"name": "6 - WatsonX SaaS services",
-			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.a8504fac-eb37-4aa0-a284-f26e876b5750-global"
+			"version_locator": "1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc.9932d3b1-369e-4b5c-ad39-9e10149b83b0-global"
+		},
+		{
+			"inputs": [
+				{
+					"name": "use_existing_resource_group",
+					"value": true
+				},
+				{
+					"name": "resource_group_name",
+					"value": "ref:../../members/1 - Account Infrastructure Base/outputs/workload_resource_group_name"
+				},
+				{
+					"name": "region",
+					"value": "ref:../../inputs/region"
+				},
+				{
+					"name": "prefix",
+					"value": "ref:../../inputs/prefix"
+				},
+				{
+					"name": "plan",
+					"value": "platinum"
+				},
+				{
+					"name": "elasticsearch_version",
+					"value": "8.12"
+				},
+				{
+					"name": "existing_kms_instance_crn",
+					"value": "ref:../../members/2a - Security Service - Key Management/outputs/kms_instance_crn"
+				},
+				{
+					"name": "kms_endpoint_type",
+					"value": "private"
+				},
+				{
+					"name": "member_host_flavor",
+					"value": "b3c.4x16.encrypted"
+				},
+				{
+					"name": "service_credential_names",
+					"value": {
+						"elastic_db_admin" : "Administrator",
+						"wxasst_db_user" : "Editor",
+						"toolchain_db_user" : "Editor"
+					}
+				}
+			],	
+			"name": "7 - Databases for Elasticsearch",
```

Comment:
Should this be renamed `"3b - Databases for Elasticsearch"` since it has a dependency on `1 - Account Infrastructure Base` and `2a - Security Service - Key Management` only (and then rename `3a - Security Service - Observability`)?

Reply:
going with this for now - will revisit later